### PR TITLE
add option -i to save as gif

### PR DESCRIPTION
**Pull request by Charlsyy**
I added the option to save as gif, however the mp4 or mov file still needs to be saved as well. I'll be trying to find a workaround, but despite that it works. When using the -i option a .gif file will be saved to the output directory

### CODE REVIEW

File: manimlib/scene/scene_file_writer.py [hunk 1]
```diff
@@ -299,10 +306,14 @@ def combine_movie_files(self):
             '-f', 'concat',
             '-safe', '0',
             '-i', file_list,
-            '-c', 'copy',
+            #'-c', 'copy',
```
Comment (eulertour) [inline manimlib/scene/scene_file_writer.py]: You should do this conditionally to avoid breaking changes.

**Comment by eulertour** [comment]
It seems strange that changing the extension alone should create create a .gif. Can you link a source that confirms this?

File: manimlib/scene/scene_file_writer.py (hunk 1 above)
Comment (Charlsyy) [reply manimlib/scene/scene_file_writer.py]: Apparently ffmpeg can't can't convert to .gif with this option enabled, that's why I removed it.

**Comment by Charlsyy** [comment]
> It seems strange that changing the extension alone should create create a .gif. Can you link a source that confirms this?

Ffmpeg does exactly that. It recognises the extension and converts it into that format
I also tested that code and it worked just fine. Here is one article showing how to create .gif files with ffmpeg 

https://engineering.giphy.com/how-to-make-gifs-with-ffmpeg/

File: manimlib/scene/scene_file_writer.py (hunk 1 above)
Comment (eulertour) [reply manimlib/scene/scene_file_writer.py]: That's fine, but you should only remove it if the `-i` flag was passed.

Comment (Charlsyy) [reply manimlib/scene/scene_file_writer.py]: Sure, I'll add that as soon as possible

**Comment by kmein** [comment]
Is there something still to be done for this PR to be merged?

**Comment by eulertour** [comment]
Nope, that reminder was enough.

**Pull request merged**